features/product/viewing_products/viewing_product_price_on_products_list.feature

```diff
@@ -33,3 +33,13 @@ Feature: Viewing a product price on products list
         When I browse products from taxon "T-Shirts"
         Then I should see the product "T-Shirt watermelon" with price "$19.00"
         And I should see "T-Shirt watermelon" product not discounted on the list
+
+    @ui @api
+    Scenario: Viewing a product with a positioned default variant
+        Given the store has a product "T-Shirt watermelon" priced at "$19.00"
+        And the product "T-Shirt watermelon" has also an "Extra Large" variant at position -1
+        And this product's price is "$10.00"
+        And the store classifies its products as "T-Shirts"
+        And this product belongs to "T-Shirts"
+        When I browse products from taxon "T-Shirts"
+        Then I should see the product "T-Shirt watermelon" with price "$10.00"
```

Comment:
I couldn't understand at first how this test ensures the variants' order is correct.
I had to look at how `this product's price is "$10.00"` is handled under the hood, in `Sylius\Behat\Context\Setup\ProductContext::theProductChangedItsPriceTo`  and saw that it uses the default variant resolver to get the variant and sets the price to it.
Now, how do you know that it actually didn't override the price for the variant priced at `$19.00`?

Reply:
I manage to add the right price on the second variant using a new `@Given` sentence :
```
@Given /^(this variant) has price at ("[^"]+")$/
```

I also use `0` instead of `-1`, setting `0` set the first created variant to the position `1` and make the second one as the default one.

Reply:
Check my comments in the other thread. I believe you can do it without adding a new method to the context.